prettyPrinters/lldb_providers.py

```diff
@@ -40,6 +40,20 @@
     from typing import Optional
 
 
+def unwrap_unique_or_non_null(unique_or_nonnull):
+    # type: (SBValue) -> SBValue
+    """
+    rust 1.33.0: struct Unique<T: ?Sized> { pointer: *const T, ... }
+    rust 1.62.0: struct Unique<T: ?Sized> { pointer: NonNull<T>, ... }
+    struct NonNull<T> { pointer: *const T }
+    """
+    ptr = unique_or_nonnull.GetChildMemberWithName("pointer")
+    if ptr.GetChildMemberWithName("pointer").IsValid():
+        return ptr.GetChildMemberWithName("pointer")
```

Comment:
Maybe it worth extracting `ptr.GetChildMemberWithName("pointer")` into variable not to call it twice?